proposals/new_policy_types.md

```diff
@@ -0,0 +1,956 @@
+      - resources:
+          kinds:
+          - Pod
+        subjects:
+        - kind: Group
+          name: dev
+```
+
+The equivalent VAP match resource would look like this:
+
+```yaml=
+spec:
+  matchConstraints:
+    resourceRules:
+      - apiGroups:   [""]
+        apiVersions: ["v1"]
+        operations:  ["CREATE"]
+        resources:   ["pods"]
+  matchConditions:
+    - name: 'check-user-groups'
+      expression: "request.userInfo.groups == ['dev']"
+```
+
+### Webhook Configuration
+
+A ValidatingWebhookConfiguration can define multiple ValidatingWebhooks, each with its own specific settings. These settings include clientConfig, rules, timeoutSeconds, failurePolicy, namespaceSelector, objectSelector, matchPolicy, and matchConditions. For example:
+
+```yaml=
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  labels:
+    webhook.kyverno.io/managed-by: kyverno
+  name: kyverno-resource-validating-webhook-cfg
+webhooks:
+- clientConfig:
+    service:
+      namespace: "example-namespace"
+      name: "example-service"
+    caBundle: <CA_BUNDLE>
+  failurePolicy: Fail
+  timeoutSeconds: 30
+  matchPolicy: Equivalent
+  matchConditions:
+    - name: 'exclude-leases'
+      expression: '!(request.resource.group == "coordination.k8s.io" && request.resource.resource == "leases")'
+  objectSelector:
+    matchLabels:
+      foo: bar
+  namespaceSelector:
+    matchLabels:
+      foo: bar
+  rules:
+  - apiGroups:
+    - '*'
+    apiVersions:
+    - '*'
+    operations:
+    - CONNECT
+    - CREATE
+    - DELETE
+    - UPDATE
+    resources:
+    - '*'
+    scope: 'Namespaced'
+```
+
+Kyverno allows users to configure some of these fields directly within a policy. For instance, timeoutSeconds, failurePolicy, and matchConditions can be set as follows:
+
+```yaml=
+spec:
+  webhookConfiguration:
+    timeoutSeconds: 30
+    failurePolicy: Ignore
+    matchConditions:
+    - name: "select-namespace"
+      expression: '(object.metadata.namespace == "cpol-fine-grained-match-conditions-ns")'
+    - name: 'exclude-requests-by-groups'
+      expression: '!("system:nodes" in request.userInfo.groups)' 
+```
+
+Kubernetes VAPs offer a `spec.matchConditions` field for fine-grained request filtering, similar to Kyverno's `preconditions`. This presents two options: utilize the VAP's matchConditions within the webhook configuration, or handle these conditions internally in the Kyverno engine.
+
+Furthermore, VAPs offer a matchPolicy field (set to `Exact` or `Equivalent`) that governs how incoming requests are matched against rules. `Exact` requires precise matching, while `Equivalent` allows for matching across different API groups or versions. 
+
+To maintain consistency and offer similar flexibility, the `ValidatingPolicy` CRD should introduce a `matchPolicy` field too. This presents two options: utilize this field within the webhook configuration, or handle it internally in the Kyverno engine.
```

Comment:
Would the `matchPolicy` in a ValidatingPolicy be propagated to validatingwebhookconfgurations?